helm/falco/templates/falco-policy-exception.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.kyvernoPolicyExceptions.enabled }}
   {{- if .Capabilities.APIVersions.Has "kyverno.io/v2alpha1/PolicyException" -}}
```

Comment:
Need to remove this

Reply:
Want to do it here or in a separate one?